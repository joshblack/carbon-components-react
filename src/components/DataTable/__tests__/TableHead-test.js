import React from 'react';
import { shallow } from 'enzyme';
import { TableHead } from '../';

describe('TableHead', () => {
  it('should render', () => {
    const wrapper = shallow(<TableHead />);
    expect(wrapper).toMatchSnapshot();
  });
});
