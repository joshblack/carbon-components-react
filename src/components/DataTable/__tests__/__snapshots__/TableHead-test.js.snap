// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TableHead should render 1`] = `<thead />`;
